src/main/java/com/iabtcf/utils/BitSetIntIterable.java

```diff
@@ -60,6 +61,14 @@ public boolean containsAll(int... source) {
         return true;
     }
 
+    @Override
+    public boolean containsAny(int... source) {
+        return Arrays
+                .stream(source)
+                .filter(value -> value < bs.size())
```

Comment:
I don't see a need for this `filter`

Reply:
logically no, but it saves us from traising `IndexOutOfBoundsException`

Reply:
as alternative I could refactor method `contains` to check if is within argument's length

Reply:
look that method is leading, you don't need to check the size of the bitSet which is wrong anyway, you would need to check if the argument is negative or not. From the Javadoc of `BitSet.get`

`@throws IndexOutOfBoundsException if the specified index is negative`